draft-ietf-acme-acme.md

```diff
@@ -3052,6 +3053,27 @@ There are certain factors that arise in operational reality that operators of
 ACME-based CAs will need to keep in mind when configuring their services.
 For example:
 
+## Key Selection
+
+ACME relies on two different classes of key pair:
+
+* Account key pairs, which are used to authenticate account holders
+* Certificate key pairs, which are used to sign and verify CSRs (and whose
+  public keys are included in certificates)
+
+Of these two types of key pair, account key pairs are far more dangerous.  While
```

Comment:
Saying a key is dangerous is weird.  "Compromise an account key has more serious consequences than compromise of a certificate key." might be a better first sentence.